.gitignore

```diff
@@ -1,4 +0,0 @@
-postgres/data
```

Comment:
Why is this being deleted? It is important to keep this as it makes sure nobody will commit the binary postgres database